src/Raven.Server/Config/Categories/MonitoringConfiguration.cs

```diff
@@ -120,5 +122,129 @@ public sealed class SnmpConfiguration : ConfigurationCategory
             [ConfigurationEntry("Monitoring.Snmp.DisableTimeWindowChecks", ConfigurationEntryScope.ServerWideOnly)]
             public bool DisableTimeWindowChecks { get; set; }
         }
+
+        [ConfigurationCategory(ConfigurationCategoryType.Monitoring)]
+        public sealed class OpenTelemetryConfiguration : ConfigurationCategory
+        {
+            [Description("Indicates if OpenTelemetry is enabled or not. Default: false")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool Enabled { get; set; }
+            
+            [Description("Indicates if server-wide OpenTelemetry is enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.ServerWideEnabled", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Consider adding a dot between ServerWide and Enabled

Reply:
Done